Sources/swift-doc/Supporting Types/Components/Relationships.swift

```diff
@@ -36,6 +36,23 @@ struct Relationships: Component {
         self.inheritedTypes = module.interface.typesInherited(by: symbol) + module.interface.typesConformed(by: symbol)
     }
 
+    var graphHTML: HypertextLiteral.HTML? {
```

Comment:
Thanks for extracting this into a property. FYI: I'll probably end up further extracting this (along with the enclosing `<figure>` tag) into a new component as part of #67.